modelina-website/src/components/playground/options/GeneralOptions.tsx

```diff
@@ -70,10 +70,28 @@ class GeneralOptions extends React.Component<
         <h3 className="text-lg font-medium leading-6 text-gray-900">
           General options
         </h3>
-        <li>
+        <li className="relative">
           <label className="flex items-center py-2 justify-between cursor-pointer">
-            <span className="mt-1 max-w-2xl text-sm text-gray-500">
+            <span className="mt-1 max-w-2xl text-sm text-gray-500 relative group flex items-center">
```

Comment:
Can this setup handle multiple groups in the same document? 🤔 I.e. hovering one item does not display all descriptions 😄